pychunkedgraph/graph/chunkedgraph.py

```diff
@@ -486,6 +486,20 @@ def is_latest_roots(
         )
         return ~np.isin(root_ids, list(row_dict.keys()))
 
+    def get_root_timestamps(
```

Comment:
we already have a `get_node_timestamps` that uses pretty much the same logic

Reply:
Thanks! Missed that somehow